app/views/layouts/mailer.text.erb

```diff
@@ -1 +0,0 @@
-<%= yield %>
```

Comment:
I'd leave this in place--someday we may _want_ a mailer layout, even though we don't use it on umaps-rails.

Reply:
It's still there. I converted it to haml in a previous commit. 2f4ffc4a6045a1fba7b391cf4d02ed5057e6ca10
This was just cleaning up extra files.